src/main/java/javaClasses/homework_4/Lilit_Manukyan/Work.java

```diff
@@ -0,0 +1,17 @@
+package javaClasses.homework_4.Lilit_Manukyan;
+
+public class Work {
+    public static void main(String[] args) {
+        Developer developer = new Developer("Arman", "Aramyan", "java");
+        QualityEngineer qualityEngineer = new QualityEngineer("Ann", "Hakobyan", "manual");
+        Staff.company = "EPAM";
+        developer.getTechstack();
+        developer.getLastName();
+        developer.getName();
+        developer.doWork();
+        Developer.getCompany();
```

Comment:
If you delete implementation of getCompany from developer class, the call will be done without any problems from the parent class.